test/k8sT/Services.go

```diff
@@ -479,6 +479,94 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sServicesTest", func() {
 		})
 	})
 
+	SkipContextIf(func() bool {
+		return helpers.RunsWithKubeProxy() || helpers.DoesNotRunOnNetNextKernel()
```

Comment:
`helpers.RunsWithKubeProxy` means kube-proxy is installed, it doesn't mean we're using it.
```suggestion
		return helpers.DoesNotRunWithKubeProxyReplacement() || helpers.DoesNotRunOnNetNextKernel()
```

Reply:
I think the original author implied RunsWithKubeProxy == does not run KPR which is AFAIK not true on GKE (we run both).

Reply:
The original author may have, but I changed that in https://github.com/cilium/cilium/pull/15543 to allow for the new kube-proxy/KPR pipeline on 4.19 :wink: